pkg/kv/kvserver/flow_control_integration_test.go

```diff
@@ -196,6 +196,15 @@ func TestFlowControlRangeSplitMergeV2(t *testing.T) {
 			h.init(mode)
 			defer h.close(makeV2EnabledTestFileName(v2EnabledWhenLeaderLevel, mode, "split_merge"))
 
+			h.comment(`-- Enable verbose logging for relevant RACv2 code files.`)
+			h.query(n1, `
```

Comment:
Do we need to print this to output? Would something like [this](https://github.com/cockroachdb/cockroach/blob/56faadf8ddb58c21678c2b4f8df5418931656887/pkg/ccl/changefeedccl/changefeed_test.go#L6756-L6757) do the trick?

Reply:
Alternative: #139484.